Ports/SDL2/patches/0001-Add-SerenityOS-platform-support.patch

```diff
@@ -495,19 +497,6 @@ index 8065a0b8cfc71e8d4dadd7d40fbe15303ce7e05d..fdebdac17e72e280fbe55fbb6e8bc7f4
  
  SDL_bool SDL_GetWindowWMInfo(SDL_Window * window, struct SDL_SysWMinfo *info)
  {
-@@ -4522,6 +4529,12 @@ int SDL_ShowMessageBox(const SDL_MessageBoxData *messageboxdata, int *buttonid)
-         VITA_ShowMessageBox(messageboxdata, buttonid) == 0) {
-         retval = 0;
-     }
-+#endif
-+#if SDL_VIDEO_DRIVER_SERENITY
-+    if (retval == -1 &&
-+        SERENITY_ShowMessageBox(messageboxdata, buttonid) == 0) {
-+        retval = 0;
-+    }
```

Comment:
Did this disappear intentionally? Why?

Reply:
SDL has refactored this part of the code, and the ShowMessageBox function call is now generic, instead of per-platform wrapped in ifdefs.